SU-SWS/saa_victoria_subtheme
# Adapt 335  webpack reorganization

# ALMOST READY - Merging on 8/6

# Summary
- Change webpack process to prepare a single css file
- Updated theme inclusion settings

# Review By (Date)
- 8/6

# Urgency
- We want to merge this between big updates for the websites and do thorough testing before and after this change.

# Steps to Test

1. pull down branch
1. run npm run publish and clear the caches
2. see a single dist/css/saa-victora.css file 
3. you should not see any visual differences, besides fixing the bright red color used on the top banner background.
4. Check very thoroughly that all the styles look correct and nothing looks missing from our styles.

# Affected Projects or Products
- n/a

# Associated Issues and/or People
- JIRA ticket ADAPT-335 for optimizing tailwind inclusion
- JIRA ADAPT-443 relies on fixing this first, then we'll continue with @apply for the fat footer styles.

We have discussed this thoroughly with @rebeccahongsf @sherakama and @imonroe and @jenbreese .
# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
